Marlin/Configuration_adv.h

```diff
@@ -1268,6 +1268,13 @@
   #define USER_GCODE_5 "G28\nM503"
 #endif
 
+/**
+ * Specify an action command to send to the host when the printer is killed.
+ * Will be sent in the form '//action:KILL_ACTION', e.g. '//action:poweroff'.
+ * The host must be configured to handle the action command.
+ */
+//#define KILL_ACTION "poweroff"
```

Comment:
Please clone these lines to the same position in all `Configuration_adv.h` files in example_configurations. The easiest way is to replace the unique lines that precede this with those same lines, plus this.